alembic/versions/879e6bc89750_add_index_to_sequencing_metrics_sample_.py

```diff
@@ -0,0 +1,29 @@
+"""Add index to sequencing metrics sample id
+
+Revision ID: 879e6bc89750
+Revises: 367ed257e4ee
+Create Date: 2023-06-14 18:09:03.849372
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "879e6bc89750"
+down_revision = "367ed257e4ee"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_index(
+        "idx_sample_internal_id",
+        "sample_lane_sequencing_metrics",
+        ["sample_internal_id"],
+        unique=False,
```

Comment:
Do you know what `unique` means here?

Reply:
I think it means that there can be multiple entries with the same `sample_internal_id` in the index. But the default is already false, so it should not be necessary to specify.

Reply:
But do we not want `unique` to be true? 🤔

Reply:
I don't think so since the `SampleLaneSequencingMetrics` are calculated per sample, flow cell and lane. So there should be many entries with the same `sample_internal_ind`! Maybe I'm confused? 😖

Reply:
Sorry, misread the table we were adding this on! You are of course correct 👍